README.rst

```diff
@@ -242,6 +251,11 @@ You will also have to manually remove a few waffle flags::
 
     tutor local run lms ./manage.py lms waffle_delete --flags account.redirect_to_microfrontend
     tutor local run lms ./manage.py lms waffle_delete --flags learner_profile.redirect_to_microfrontend
+    tutor local run lms ./manage.py lms waffle_delete --flags discussions.enable_discussions_mfe
```

Comment:
We're starting to have so many waffles that we need to disable individually for each MFE... Could you please add `# MFE <name>` comments such that we know for which MFE each waffle need to be toggled? In the case of the discussion MFE it's fairly obvious, of course.

Reply:
Good idea.  See if the changes work, then I'll apply it to the other PRs.  Or, actually, I'll just wait to group all the new MFEs in one PR, once they're all approved and #69 merges.